src/field.cpp

```diff
@@ -2374,12 +2356,25 @@ void map::monster_in_field( monster &z )
             }
             break;
 
+        case fd_fungal_haze:
+            if( !z.type->in_species("FUNGUS") &&
+                !z.type->has_flag("NO_BREATHE") &&
```

Comment:
This was always the case with fungal haze. It had "%s breathes in millions of spores" message too.